config/core/300-resources/rolloutorchestrator.yaml

```diff
@@ -98,6 +98,15 @@ spec:
                         description: TargetReplicas indicates an estimated number of replicas.
                         type: integer
                         format: int32
+                      configurationName:
+                        description: ConfigurationName of a configuration to whose latest revision we will send this portion of traffic. When the "status.latestReadyRevisionName" of the referenced configuration changes, we will automatically migrate traffic from the prior "latest ready" revision to the new one.  This field is never set in Route's status, only its spec.  This is mutually exclusive with RevisionName.
+                        type: string
+                      tag:
+                        description: Tag is optionally used to expose a dedicated url for referencing this target exclusively.
+                        type: string
+                      url:
+                        description: URL displays the URL for accessing named traffic targets. URL is displayed in status, and is disallowed on spec. URL must contain a scheme (e.g. http://) and a hostname, but may not contain anything else (e.g. basic auth, url path, etc.)
+                        type: string
```

Comment:
Do we need these fields for initialRevisions and targetRevisions?

Reply:
For targetRevisions, yes, because it will need to pick up all the info input from the user via the knative service.
StageTargetRevisions is created based on targetRevisions. So it is yes for StageTargetRevisions as well.
initialRevisions of the current service can be the targetRevisions of the previous service. so it is yes for initialRevisions to keep all the info.